Linux-Headers don't ship with include/asm/page.h anymore and it is not
needed by perl anyway.

Signed-off-by: Jonas Karlsson <jonas@gobolinux.org>

--- perl-5.8.8/ext/IPC/SysV/SysV.xs.no_asm_page_h       2001-06-30 14:46:07.000000000 -0400
+++ perl-5.8.8/ext/IPC/SysV/SysV.xs     2006-06-02 17:37:22.000000000 -0400
@@ -3,9 +3,6 @@
 #include "XSUB.h"

 #include <sys/types.h>
-#ifdef __linux__
-#   include <asm/page.h>
-#endif
 #if defined(HAS_MSG) || defined(HAS_SEM) || defined(HAS_SHM)
 #ifndef HAS_SEM
 #   include <sys/ipc.h>
